test/acceptance/features/steps/primazacluster.py

```diff
@@ -133,3 +134,9 @@ def ensure_application_namespace_exists(
         context, cluster_name: str, namespace: str, cluster_environment: str, tenant: str = "primaza-system"):
     primaza = context.cluster_provider.get_primaza_cluster(cluster_name)  # type: PrimazaCluster
     primaza.create_application_namespace(namespace, tenant, cluster_environment)
+
+
+@step(u'On Primaza Cluster "{cluster_name}", sleep for {time_interval:d} minutes')
+def ensure_to_sleep_for_certain_time_interval(context, cluster_name: str, time_interval: int):
+    context.cluster_provider.get_primaza_cluster(cluster_name)
+    time.sleep(time_interval*60)
```

Comment:
we already have one step like this at https://github.com/primaza/primaza/blob/ec14c4d84e0ae53c3e3fbf683da003f2f4abde62/test/acceptance/features/steps/util.py#L31

Reply:
Done